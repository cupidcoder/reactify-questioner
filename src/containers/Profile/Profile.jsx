import React from 'react';
import BaseLayout from '../BaseContainer/BaseContainer';
import UserStatHeader from '../../components/UserStatHeader/UserStatHeader';
import UserStat from '../../components/UserStat/UserStat';
import UserStatFeed from '../../components/UserStatFeed/UserStatFeed';
import style from './Profile.css';

const Profile = () => {
  const UserDetails = {
    name: 'John Doe',
    joined: 'January 31st, 2019'
  };
  return (
    <BaseLayout>
      <div className={style.Profile}>
        <UserStatHeader {...UserDetails} />
        <UserStat />
        <UserStatFeed />
      </div>
    </BaseLayout>
  );
};
export default Profile;
