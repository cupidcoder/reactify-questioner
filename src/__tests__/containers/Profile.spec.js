import React from 'react';
import { shallow } from 'enzyme';
import Profile from '../../containers/Profile/Profile';

describe('Profile', () => {
  it('Should correctly save snapshot of the Profile page', () => {
    const wrapper = shallow(<Profile />);
    expect(wrapper).toMatchSnapshot();
  });
});
