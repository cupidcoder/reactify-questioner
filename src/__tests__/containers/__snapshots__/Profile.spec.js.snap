// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Should correctly save snapshot of the Profile page 1`] = `ShallowWrapper {}`;
